config/routes.rb

```diff
@@ -21,6 +21,7 @@
     get 'support'
     get 'terms'
     get 'dashboard'
+    get 'site-changes'
```

Comment:
`get 'site-changes', to: :site_changes` should allow you to maintain the desired path but use underscoring for the controller method.